docs/docsite/rst/community/communication.rst

```diff
@@ -143,6 +143,7 @@ Many of them meet in chat. If you want to get involved in a working group, join
 - `Diversity Working Group <https://github.com/ansible/community/wiki/Diversity>`_ - Matrix: `#diversity:ansible.com <https://matrix.to:/#/#diversity:ansible.com>`_ | IRC: ``#ansible-diversity``
 - `Docker Working Group <https://github.com/ansible/community/wiki/Docker>`_ - Matrix: `#devel:ansible.com <https://matrix.to:/#/#devel:ansible.com>`_ | IRC: ``#ansible-devel``
 - `Documentation Working Group <https://github.com/ansible/community/wiki/Docs>`_ - Matrix: `#docs:ansible.com <https://matrix.to:/#/#docs:ansible.com>`_ | IRC: ``#ansible-docs``
+- `Execution Environments Group <https://forum.ansible.com/g/ExecutionEnvs>`_
```

Comment:
Would it make sense to include `https://matrix.to/#/#users:ansible.com` here too?

Reply:
maybe later, will see. at least now, i don't think someone from current members hang out there, so to avoid frustration let's keep it in the forum